lib/game.rb

```diff
@@ -37,4 +37,13 @@ def initialize(params)
     @venue_link = params[:venue_link]
   end
 
+  def total_score
```

Comment:
These two aren't specifically in interaction pattern. Using them as helper for methods at Stat Tracker.